rbi/gems/didyoumean.rbi

```diff
@@ -24,6 +24,9 @@ end
 module DidYouMean::JaroWinkler
   THRESHOLD = T.let(T.unsafe(nil), Float)
   WEIGHT = T.let(T.unsafe(nil), Float)
+
+  sig { params(str1: String, str2: String).returns(T.any(Float, Integer)) }
```

Comment:
Why `T.any(Float, Integer)` instead of just `Float`?

Reply:
For the cases that return `0` (on mobile, but "foo" vs. "bar" should repro).

Reply:
Just confirming:
```
> DidYouMean::JaroWinkler.distance('foo', 'bar').class
=> Integer
```